Code/ww3d2/dx8wrapper.h

```diff
@@ -794,10 +794,10 @@ WWINLINE void DX8Wrapper::Clamp_Color(Vector4& color)
 
 WWINLINE unsigned int DX8Wrapper::Convert_Color(const Vector3& color,float alpha)
 {
-#if defined(_M_IX86)
 	const float scale = 255.0;
 	unsigned int col;
 
+#if defined(_MSC_VER) && defined(_M_IX86)
```

Comment:
I'd suggest just removing the ASM version entirely and use the C++ version in all builds.

Reply:
Done. Please review.